main.go

```diff
@@ -76,7 +89,7 @@ func (c CFDrainCLI) GetMetadata() plugin.PluginMetadata {
 				Name:     "drain",
 				HelpText: "Creates a user provided service for syslog drains and binds it to a given application.",
 				UsageDetails: plugin.Usage{
-					Usage: "drain <app-name> <syslog-drain-url> [options]",
+					Usage: "drain APP_NAME SYSLOG_DRAIN_URL [OPTIONS]",
```

Comment:
Is this a standard used elsewhere? Looking at things like `man tail` and `man grep`, it looks like we should do `drain [app name] [options]`.

Reply:
The cf cli uses this format. That's why I changed it.

Reply:
This convention is used elsewhere in the cf_cli. So I changed it to match.